Auth0/Auth0WebAuth.swift

```diff
@@ -33,6 +36,11 @@ final class Auth0WebAuth: WebAuth {
     private(set) var overrideAuthorizeURL: URL?
     private(set) var provider: WebAuthProvider?
     private(set) var onCloseCallback: (() -> Void)?
+    
+    #if os(iOS)
+    private(set) var useWebViewProvider = false
+    private(set) var webViewProviderPresentationStyle: UIModalPresentationStyle = .fullScreen
+    #endif
```

Comment:
Using a func like https://github.com/auth0/Auth0.swift/blob/master/Auth0/SafariProvider.swift#L37 also has the advantage of being on its own (gated) file, so there's no need to have these conditionals here.

Reply:
changed accordingly